api/python/provisioner/config.py

```diff
@@ -184,6 +184,14 @@
     'replace_node'
 ]
 
+
+# List of commands for which we purposely reduce log level severity to suppress
+# logging noise and prevent log files size growing.
+LOG_SUPPRESSION_CMDS = [
```

Comment:
Please add here all get commands that we  have currently

Reply:
Done